core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -435,20 +436,167 @@ ClientRequestContext newDerivedContext(RequestId id, @Nullable HttpRequest req,
      */
     long responseTimeoutMillis();
 
+    /**
+     * Clears the response timeout.
+     */
+    void clearResponseTimeout();
+
     /**
      * Sets the amount of time allowed until receiving the {@link Response} completely
      * since the transfer of the {@link Response} started. This value is initially set from
```

Comment:
How about this:

```
Schedules the response timeout that is triggered when the {@link Response} is not 
fully received within the specified amount of time since the {@link Response} started.
```